x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -40,25 +41,56 @@ FIPS 140-2 JVM. In this case, you would need to also regenerate your
 necessary configuration changes outlined below, before starting each node.
 
 [float]
+[[fips-140-compliance-es]]
 === Configuring {es} for FIPS 140-2
 
 Apart from setting `xpack.security.fips_mode.enabled`, a number of security
 related settings need to be configured accordingly in order to be compliant
 and able to run {es} successfully in a FIPS 140-2 enabled JVM.
 
 [float]
+[[fips-140-compliance-tls]]
 ==== TLS
 
 SSLv2 and SSLv3 are not allowed by FIPS 140-2, so `SSLv2Hello` and `SSLv3` cannot
-be used for <<ssl-tls-settings, `ssl.supported_protocols`>>
-
-NOTE: The use of TLS ciphers is mainly governed by the relevant crypto module
-(the FIPS Approved Security Provider that your JVM uses). All the ciphers that
-are configured by default in {es} are FIPS 140-2 compliant and as such can be
-used in a FIPS 140-2 JVM. (see <<ssl-tls-settings, `ssl.cipher_suites`>>)
+be used for <<ssl-tls-settings,`ssl.supported_protocols`>>
+
+The use of TLS ciphers is mainly governed by the relevant crypto module
+(the FIPS Approved Security Provider that your JVM uses). In general, the
+ciphers that are configured by default in {es} are FIPS 140-2 compliant and can
+be used in a FIPS 140-2 JVM. If you use JDK 11, however, the following ciphers
```

Comment:
This is only true when users configure their JVM in FIPS 140 mode with the help of BouncyCastle crypto and JSSE providers so I don't think that we want to make a blanket statement about the list of supported ciphers